frontend/src/features/books/BookDetails.jsx

```diff
@@ -1,27 +1,22 @@
 import React, { useEffect, useState } from 'react'
```

Comment:
@nshelokar-vf, add `;` here

Do we need to import `React` here? Please check this